cpp/src/gandiva/gdv_function_stubs.cc

```diff
@@ -794,6 +795,54 @@ const char* gdv_fn_initcap_utf8(int64_t context, const char* data, int32_t data_
   *out_len = out_idx;
   return out;
 }
+
+GANDIVA_EXPORT
+const char* gdv_fn_elt_utf8(int64_t context, int32_t pos, const char* data,
```

Comment:
elt is a function with variable number of arguments - there is no expectation for the arguments to be concatenated using a ',' separator. This code assumes that

Please implement elt(int, string1, string2) - elt that works on 2 string arguments and
elt(int, string1, string2, string3) and elt(int, string1, string2, string3, string4) and elt(int, string1, string2, string3, string4, string5)

If the query has more arguments, the query needs to be modified

Reply:
It was the only way that I've found to simulate a variadic function in Gandiva, but I changed for your approach.